product-scenarios/5-route-messages-between-systems/pom.xml

```diff
@@ -35,6 +35,7 @@
     <modules>
         <module>05-synapseConfigProject</module>
         <module>5.3-load-balance-messages-among-systems</module>
+        <module>5.1-Route-based-on-the-content-of-the-messages</module>
```

Comment:
Shall we rearrange the order of modules according to scenario number?

Reply:
Done